src/components/ElevationSlider.vue

```diff
@@ -35,68 +35,61 @@ function roundToNearest100 (x: number) {
   }
 })
 export default class ElevationSlider extends Vue {
-  @Prop({ default: 0 }) value!: number
+  @Prop({ default: 1 }) value!: number
   @Prop({ default: 0 }) minValue!: number
-  @Prop({ default: 0 }) maxValue!: number
+  @Prop({ default: 10 }) maxValue!: number
 
```

Comment:
Shall we have default -10 to 0? with default value -1?
We know that the elevation is always negative.